tests/cython/atomspace/test_tensortv.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+
+from opencog.bindlink import execute_atom
+from opencog.atomspace import AtomSpace, types
+from opencog.utilities import initialize_opencog, finalize_opencog
+from opencog.type_constructors import *
+
+
+torch_found = True
+try:
+    from opencog.atomspace import TensorTruthValue
+    from torch import tensor
+    import torch
+    TTruthValue = TensorTruthValue
```

Comment:
Why did you not renamed classes in test? As test can be used as example of code it can be misleading.

Reply:
I don't see misleading class names in this file. Which one looks misleading to you?